browser/ui/tabs/brave_split_tab_menu_model.cc

```diff
@@ -62,6 +75,15 @@ std::u16string BraveSplitTabMenuModel::GetLabelForCommandId(
     return l10n_util::GetStringUTF16(IDS_IDC_BREAK_TILE);
   }
 
+  if (id == CommandId::kToggleLinkState) {
+    const split_tabs::SplitTabId split_id = GetSplitTabId();
+    split_tabs::SplitTabData* const split_tab_data =
+        tab_strip_model_->GetSplitData(split_id);
+    const bool linked = split_tab_data->linked();
```

Comment:
Maybe having bool AreSplitTabLinked(SplitTabId, TabStripModel*) would be good for deduplicate logic.